master/getting-started/kubernetes/tutorials/simple-policy.md

```diff
@@ -67,8 +82,11 @@ EOF
 This will prevent all access to the nginx Service.  We can see the effect by trying to access the Service again.
 
 ```
-# Run a Pod and try to access the `nginx` Service.
-$ kubectl run --namespace=policy-demo access --rm -ti --image busybox /bin/sh
+kubectl run --namespace=policy-demo access --rm -ti --image busybox /bin/sh
+```
+It should return something like the following.
```

Comment:
another one - "open a shell within the pod"

and, needs to split out the `wget` into a copy-able box.